samples/DLLMethodCaller/DLLMethodCallerEA.mq4

```diff
@@ -41,36 +41,59 @@
 	int nquotes_get_property_array_size(string name);
 	int nquotes_get_property_adouble(string name, double& value[]);
 #import
+//extern int MoneyAmount = 12345;
+//extern datetime MyBirthday;
+//extern bool AmMarried;
```

Comment:
I think it is a good idea to have these variables and use them in the code somehow, but if you don't use them - better to remove the comment.